Resolved bug 1327 : Design System > Components >Dialog > Default > Cross button alignment issue.

nitinsonawane123 opened the pull request:
## Description
> Resolve the issues on Components: 
1. Resolved issue on Components - Polls-Option - Plus button alignment issue.

## Screenshots :
 
![image](https://github.com/user-attachments/assets/bba0d477-142f-455e-bd55-f7b43f5f7c78)

 
## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

alok-nakhate reviewed (approved):
Changes are reviewed and approved.

nitinsonawane123 committed:
Commit Changes for bug 1327

nknikhilkasar committed:
Merge branch 'development' into users/nitin/bug-1327-component-dialog-cross-button-alignment-issue

nknikhilkasar commented:
Add validation for polls option if user don't add any text into input field he will not able add any option
![image](https://github.com/user-attachments/assets/c1f8cf47-658d-4b9d-a02b-e1d146e60607)

 not able added other polls option except added last one pls fixed these issue.
![image](https://github.com/user-attachments/assets/cecf967e-12c0-4d78-adfa-2af786abc70b)

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/bug-1327-component-dialog-cross-button-alignment-issue

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/bug-1327-component-dialog-cross-button-alignment-issue

nitinsonawane123 committed:
Commit Changes for bug 1327

nitinsonawane123 commented:
Resolved.
- Added validation for polls option if user don't add any text into input field he will not able add any option.
- fixed alignment issue small, large & tablet mobile view.

![image](https://github.com/user-attachments/assets/455db5a9-d9a9-4dfa-a727-c956f8380cfa)
![image](https://github.com/user-attachments/assets/58b7d4a6-be33-4b02-8144-1c418928a75e)
![image](https://github.com/user-attachments/assets/4093c237-c43d-46d8-8223-82c42fa97a76)
![image](https://github.com/user-attachments/assets/74c0fd9e-b9e8-4c4e-9cb1-ef42e30d9b27)

supriyartambewaiin commented:
@nknikhilkasar, the second screenshot which you mention here is not related to this bug. that is another bug for all data table component. @rameshwarkamandar created another bug for it. @nitinsonawane123 is going to work on it.

nitinsonawane123 committed:
Commit Changes for bug 1327

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/bug-1327-component-dialog-cross-button-alignment-issue

supriyartambewaiin reviewed (approved):
Reviewed and Approved
